app/javascript/styles/components.scss

```diff
@@ -3507,17 +3504,22 @@ button.icon-button.active i.fa-retweet {
 
 .actions-modal {
   .status {
+    background: $white;
+    border-bottom-color: $ui-secondary-color;
+    padding-top: 10px;
+    padding-bottom: 10px;
     overflow-y: auto;
     max-height: 300px;
   }
 
-  max-height: 80vh;
-  max-width: 80vw;
-
   ul {
-    overflow-y: auto;
     flex-shrink: 0;
 
+    @media screen and (max-height: 430px) {
+      flex-shrink: 1;
+      overflow-y: auto;
```

Comment:
I thought this style can be set always (outside of media query), but it doesn't show scrollbar on portrait -> landscape changing with that way. So I will set this only if needed.